README.md

```diff
@@ -158,14 +168,35 @@ relationships between the two are set up in Active Record.
 Write code to do the following:
 
 1. Create a lamp with 3 wishes remaining and a genie named 'Genie'
+
 2. Create a relationship between 'Genie' and the lamp.
+
 3. Update the lamp so it only has one wish left.
   * Oh no... Jafar has Aladdin! Thankfully he's wished to become a genie himself!
+
 4. Create a new Genie named 'Jafar' and put him in a new lamp with 3 wishes left.
+
 5. Free the good Genie by setting his lamp to `nil`
 
 
 Write your code here:
-```ruby
-# code here
+```
+
+myLamp = Lamp.create(wishe_remaining: 3)
+myGenie = Genie.create(name: 'Genie')
+
+class Genie < ActiveRecord::Base
+  belongs_to :lamp
```

Comment:
For a one to one relationship, one of the models needs to be `has_one ` depending on where you want a foreign key 
For example:
`has_one :lamp` and not `belongs_to :lamp`